### PR TITLE
WIP: ECIP-1000: Irregular Process Transitions (Citations version)

### PR DESCRIPTION
**Pull request by meowsbits**
Based on #220.

The intention of commits  https://github.com/ethereumclassic/ECIPs/commit/164071a9ff07499edbe2c042772ab9e188fa7c53, https://github.com/ethereumclassic/ECIPs/commit/2784614c62823b0df2ff8385f0113d0e917d93cb, and https://github.com/ethereumclassic/ECIPs/commit/219c88f91eb5737bfa5fbc769b54073923c13108 are to cite the bespoke subject matter for convenient and specific reference.

My intention is limited strictly to using primary source citations as a self-evidencing account of events.

I have used Github metadata included on [branch: gh](https://github.com/ethereumclassic/ECIPs/tree/gh) as sources for citations. This location can be considered tentative and _adhoc_, with the specific location of the cited data of no importance. (I'm happy to get rid of this branch on this remote if it bothers anybody, but there are a lot of other seemingly-"junk" branches here as well.) An equivalent branch and data set can be found at https://github.com/meowsbits/ECIPs/tree/gh. With #235 I have proposed adding this data and possibly maintenance processes for it in a more permanent way.

---

### TODO

- [ ] Finish citation for the following clause:

  > `This constituted a process deviating from the then-established [procedural specification](TODO).`

In [ECIP-1000 #Contributing > Tips](https://github.com/ethereumclassic/ECIPs#tips) I found:

> - For a Standards Track ECIP of type Core, ask to have your issue added to the agenda of an upcoming core developer team meeting. There, it can be discussed for inclusion in a future hard fork. If developers agree to include it, the ECIP editors will update the state of your ECIP to 'Accepted'.
> - For all other ECIPs, open a PR changing the state of your ECIP to 'Final'. An editor will review your draft and ask if anyone objects to its finalization. If the editor decides there is no rough consensus - for instance, because contributors point out significant issues with the ECIP - they may close the PR and request that you fix the issues in the draft before trying again.

In [ECIP-1000 #ECIP Status Terms](https://github.com/ethereumclassic/ECIPs#ecip-status-terms) I found: 

> Accepted - An ECIP that has been in Last Call for at least 2 weeks and any technical changes that were requested have been addressed by the author.

In [ECIP-1000 #Process Overview](https://github.com/ethereumclassic/ECIPs#process-overview) I found: 

> Having an ECIP within the folder of the repository does not make it a formally accepted standard until its status becomes "Final". For an ECIP to become Final it requires the common consent of the ecosystem. Those proposing changes should consider that, ultimately, consent may rest with the consensus of the Ethereum Classic implementers and users.

These handle `Accepted` and `Final` status merge qualifications, but I can't seem to find the right specifications to fit our needs for the deviant `Last Call` status merge.  @sorpaas Can you please help with this?

### CODE REVIEW

File: _specs/ecip-1000.md [hunk 1]
```diff
@@ -355,9 +355,9 @@ In addition, it is recommended that literal code included in the ECIP be dual-li
...
 
-* In [PR#199](https://github.com/ethereumclassic/ECIPs/pull/199), Afri Schoedon merged a PR moving ECIP-1061 to "Last Call" status. At the time, Afri dismissed a review of clear objections from ECIP editor Wei Tang, without waiting for consultations of the community, or resolutions with the review submitter. We here declare that this particular action from Afri followed the ECIP process. 
+* [Pull Request #199](https://github.com/ethereumclassic/ECIPs/blob/gh/.gh-pullrequests/00199.json) was [merged](https://github.com/ethereumclassic/ECIPs/blob/gh/.gh-pullrequests/00199_event_2837817387.json), [moving it's associated ECIP to `Last Call` status](https://github.com/ethereumclassic/ECIPs/blob/gh/.gh-pullrequests/00199.patch#L21) after [dismissing](https://github.com/ethereumclassic/ECIPs/blob/gh/.gh-pullrequests/00199_event_2837816513.json) a [review](https://github.com/ethereumclassic/ECIPs/blob/gh/.gh-pullrequests/00199_review_323755348.json) which [provided a reasonable objection](https://github.com/ethereumclassic/ECIPs/blob/gh/.gh-pullrequests/00199_review_323755348.json#L24). This constituted a process deviating from the then-established [procedural specification](TODO).
```
Comment (meowsbits) [inline _specs/ecip-1000.md]: > This constituted a process deviating from the then-established [procedural specification](TODO).

Needs to be finished. See https://github.com/ethereumclassic/ECIPs/pull/237#issue-348371179 for WIP reasoning.

**Comment by sorpaas** [comment]
@meowsbits The main violation was the part "dismissing the review from another editor". That's rather a violation of editing responsibilities for Github workflow, which is implicitly covered in this sentence, in *ECIP Editor Responsibilities & Workflow*.

> ECIP editors are intended to fulfill administrative and editorial responsibilities. ECIP editors monitor ECIP changes, and update ECIP headers as appropriate.

The section, also confirms that, PRs must be merged only when it's ready:

> [Once the ECIP is ready,] the ECIP editor will -- merge the pull request when it is ready.

Having a reasonable request change from another editor is clearly not a status when a PR is ready.

In addition, the action also violates this paragraph in *ECIP Status Field* section:

> An ECIP may only change status from Draft (or Rejected) to Last Call, when the author deems it is complete, has a working implementation (where applicable), and has community plans to progress it to Final status.

We did not have community plans to move ECIP-1061 forward at the moment the PR was merged, because in the last core dev meeting the consensus was on ECIP-1072, and only much later changed to ECIP-1061. That's also what the request-change review was about, but got dismissed.

**Comment by husainfazel** [comment]
"because in the last core dev meeting the consensus was on ECIP-1072"

https://github.com/ethereumclassic/ECIPs/pull/220#issuecomment-560015672
https://github.com/ethereumclassic/ECIPs/pull/220#issuecomment-560017107

**Comment by sorpaas** [comment]
@husainfazel As I said previously, there are several fatal logic inconsistencies if the last dev call consensus indeed reached on ECIP-1061. You can read more about that here https://github.com/ethereumclassic/ECIPs/issues/177#issuecomment-560057744

Note that so far no one has been able to provide any objections to those logic inconsistencies of ECIP-1061.

The first comment you posted also misread an important part of ECIP-1000:

> We try to build consensus around an ECIP, but if that’s not possible, you can always submit a competing ECIP.

Again, I don't think anyone has been able to provide valid reasons to object the last dev meeting's decision -- ECIP-1072. If you have one, let me know, otherwise, please don't spam the repo any more.

**Comment by husainfazel** [comment]
I read your comments. You do realize that several days later you are producing a set of note that corroborate your version of events, had you posted a full set of notes for the entire meeting right at the start, you'll find people less likely to dispute the veracity of those notes/your recollection of the meeting. @shanejonas has a different view and without a recording of the conversation, no-one can say who is correct.

I won't post anymore because I'm an outsider to this process and I don't want to antagonize anyone but you CANNOT claim community consensus that your ECIP was the one everyone settled on when there's more than 1 person that attended the meeting claiming otherwise!

**Comment by sorpaas** [comment]
@husainfazel You also cannot claim that you have consensus on ECIP-1061 when more than 1 person attended meeting claiming otherwise. That is the solid foundation for why the action moving ECIP-1061 to "Last Call" is invalid and violated the ECIP process. And that's the solid foundation for why we need irregular process transitions.

The logic inconsistency I made was not based on my note, it was based on Afri's note, which was published right away after the meeting ends. No one has able to dispute those logic inconsistencies, and that makes it a strong evidence that the decision reached was ECIP-1072.

**Pull request closed without merging**